nx/lib/nx.ex

```diff
@@ -12084,6 +12084,80 @@ defmodule Nx do
     apply(impl!(tensor), kind, [out, tensor, opts])
   end
 
+  @doc """
+  Creates a tensor of shape `{n}` with linearly spaced samples between `start` and `stop`.
+
+  ## Options
+
+    * `:n` - The number of samples in the tensor.
+    * `:axis_name` - Optional name for the output axis.
+    * `:type` - Optional type for the output. Defaults to `{:f, 32}`
+    * `:endpoint` - Boolean that indicates whether to include `stop`
```

Comment:
Maybe this should be "inclusive" or "include_stop"? Endpoint isn't 100% clear imo

Reply:
I like inclusive although I think numpy uses endpoint.

Reply:
What about include_endpoint?